internal/core/run_clang_format.sh

```diff
@@ -6,8 +6,15 @@ else
 fi
 CorePath=$1
 
+# Check if clang-format-10 exists
+if command -v clang-format-10 >/dev/null 2>&1; then
```

Comment:
I am not sure whether we can replace all the `clang-format-10` with `clang-format` in the codebase. So I write a simple check here.